python/pyspark/sql/functions.py

```diff
@@ -8159,6 +8159,382 @@ def to_number(col: "ColumnOrName", format: "ColumnOrName") -> Column:
     return _invoke_function_over_columns("to_number", col, format)
 
 
+@try_remote_functions
+def char(n: "ColumnOrName") -> Column:
```

Comment:
i suggest we use `col: "ColumnOrName"` for similar cases in python

Reply:
e.g. https://github.com/apache/spark/blob/79100ce225d8fafe58f8f61329e4f0ef51c2acf9/python/pyspark/sql/functions.py#L7268-L7339

Reply:
+1

Reply:
OK